src/stickybits.js

```diff
@@ -164,13 +164,17 @@ class Stickybits {
       - offset = number
       - stickyStart = number
       - stickyStop = number
+      - spacer = dom element | null
```

Comment:
This should be controlled by the plugin. 

- It should go write into `position: fixed` code (not optional). 
- It should provide a class `stickybits-item-stub` or or something like that. 
- The inline css of the stub should have a height set to the calculated stickybit item height. 
- The stub should be set to `display:block;` vs `display:none` when the stickybit item is set to `position: fixed`. 

See [Fixed Sticky](https://github.com/filamentgroup/fixed-sticky) for reference.

Reply:
@yowainwright Thank you for comments 👍 Let me discuss before fixing.

> - It should go right into position: fixed code (not optional).

I agree that in terms of minimizing responsibility of instance. However I feel a bit uncertain that stickybits has to get stub element reference from current DOM tree again when destroying instance. 
[fixed-sticky does so](https://github.com/filamentgroup/fixed-sticky/blob/05660f1b8721f4cd9dd7e1703103de84847093cb/fixedsticky.js#L147
) , and it can lose the reference when re-getting. ( I know it's very rare case, but possible because DOM tree should be regarded as a global variable) 

> - It should provide a css class `stickybits-item-stub` , or something like that.

OK, I'll add it after the other discussion!

> - The inline css of the stub should have a height set to the calculated stickybit item height.
> - The stub should be set to `display:block;` vs `display:none` when the stickybit item is set to `position: fixed` (but I know it is a bit more nuanced than that 😄).

They seem to be implemented here, could you tell me if it is different from you intended ?
https://github.com/yowainwright/stickybits/blob/cdc8a8503a87e256e6da4f03c4c576fa3e1c0ebc/src/stickybits.js#L381-L390

Note: `spacerDisplay` is assumed it can be `block` or something else, such as `table-header-group` .

Reply:
Sorry @ykdr2017, I was just writing out instructions. 😃 

As this issue supports a very small (but important use-case) was to ensure that the footprint that this update is small.